package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@koopjs/koop-provider-hub-search",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Comment:
version numbers should not get bumped on the PR.  They get bumped in a "release-commit" on master branch.

Reply:
Reverted the version. it must have gotten there from previous commits.